03/app.js

```diff
@@ -0,0 +1,26 @@
+const init = function() {
+
+const btnsList = document.querySelectorAll('button');
+// console.log(btnsList);
+
+const addText = function(event) {
+
+    if(event){
+    this.innerText = 'clicked';
+    console.log('clicked');
+    
+    // console.log(event.currentTarget.indexOf);
+    //?dev trochę nie wiem jak to zrobić gdybym dokładnie w tym miejscu chciał w konsoli wyświetlać 
```

Comment:
Musiałbyś zawrzeć tą informacje w html np. `<button data-number="1">` i potem pobierać tą informacje w JS tj. `this.dataset.number`